### PR TITLE
Add community.asa to galaxy via zuul.d projects.yaml

### PR DESCRIPTION
**Pull request by justjais**
Add community.asa to galaxy via zuul.d projects.yaml

### CODE REVIEW

File: zuul.d/projects.yaml [hunk 1]
```diff
@@ -96,6 +96,19 @@
       - integrated-queue
       - publish-to-galaxy
 
+- project:
+    name: github.com/ansible-collections/community.asa
+    third-party-check:
```
Comment (pabelanger) [inline zuul.d/projects.yaml:101]: lets gate this project with zuul, or is there an issue doing that?

Comment (justjais) [reply zuul.d/projects.yaml:105]: +1

Comment (pabelanger) [reply zuul.d/projects.yaml:105]: okay, this is not correct then. We need to remove this, and use the folowing
```
- project:
    name: github.com/ansible-collections/community.asa
    merge-mode: squash-merge
    templates:	    templates:
      - system-required
      - publish-to-galaxy

Comment (justjais) [reply zuul.d/projects.yaml]: cool, I have made the changes

**Pull request merged**